libs/community/langchain_community/vectorstores/supabase.py

```diff
@@ -333,10 +336,10 @@ def _add_vectors(
                 "content": documents[idx].page_content,
                 "embedding": embedding,
                 "metadata": documents[idx].metadata,  # type: ignore
+                **kwargs,
```

Comment:
out of curiosity what's value of inserting same thing in every row? do you have an example of how this would be used?

Reply:
I need to content from multiple GitHub repositories and embed it into a database. For version control, it is necessary to record the commit Id and commit sha. So, I'm thinking of adding some extra fields for this purpose. Here's how I plan to use it:
```
SupabaseVectorStore.from_documents(
            docs,
            embeddings,
            client=get_client(),
            table_name=TABLE_NAME,
            query_name=QUERY_NAME,
            chunk_size=CHUNK_SIZE,
            repo_name=repo_name,
            commit_id=commit_id
            commit_sha=commit_sha
        )
```
Result table：
![image](https://github.com/langchain-ai/langchain/assets/10885578/90163383-39bf-4d4c-91a2-17689a2293ae)